frontend/src/modules/dashboard/containers/index.ts

```diff
@@ -1,3 +1,15 @@
+/**
+ * @summary A short one-line description for the file
```

Comment:
Originally the `containers` folder was meant for containers in the Redux sense, i.e. things which connected generic React components to specific Redux selectors. However, `Controlnfo`/`ValueInfo`/`LargeValueInfo` don't seem to match that pattern. Is there a better place to move those files to?

Reply:
Yes, in that case,  `Controlnfo`/`ValueInfo`/`LargeValueInfo`  can be moved to `views` folder